addons/csi-vmware-cloud-director/driver.yaml

```diff
@@ -201,16 +263,6 @@ metadata:
   name: vcloud-csi-configmap
   namespace: kube-system
 ---
-apiVersion: v1
-data:
-  password: '{{ with .Credentials.VCD_PASSWORD }}{{ . | b64enc }}{{ end }}'
-  refreshToken: '{{ with .Credentials.VCD_API_TOKEN }}{{ . | b64enc }}{{ end }}'
-  username: '{{ with .Credentials.VCD_USER }}{{ . | b64enc }}{{ end }}'
-kind: Secret
-metadata:
-  name: vcloud-basic-auth
-  namespace: kube-system
----
```

Comment:
@kron4eg this is a duplicate of https://github.com/kubermatic/kubeone/blob/main/addons/csi-vmware-cloud-director/vcloud-basic-auth.yaml